front/components/assistant/conversation/ConversationViewer.tsx

```diff
@@ -289,8 +294,9 @@ const ConversationViewer = React.forwardRef<
             void mutateConversations(); // to refresh the list of convos in the sidebar (title)
             break;
           case "agent_message_done":
-            // Immediate mark as read and do not mutate the list of convos in the sidebar to avoid any network request.
-            void markAsRead(conversationId, false);
+            // Mark as read and do not mutate the list of convos in the sidebar to avoid any network request.
+            // Debounce the call as we might receive multiple events for the same conversation (as we replay the events).
```

Comment:
debounce waits 2s or does the call at first event and then waits for 2s ?

Asking because 2s might appear quite slow (already feels a bit slow now?)